src/pybind/mgr/dashboard/frontend/e2e/block/mirroring.po.ts

```diff
@@ -1,5 +1,52 @@
+import { $, browser, by, element } from 'protractor';
+import { Helper } from '../helper.po';
 import { PageHelper } from '../page-helper.po';
 
 export class MirroringPageHelper extends PageHelper {
   pages = { index: '/#/block/mirroring' };
+
+  // Goes to the mirroring page and edits a pool in the Pool table. Clicks on the
+  // pool and chooses a option (either pool, image, or disabled)
+  editMirror(name, option) {
+    this.navigateTo();
+    // Clicks the pool in the table
+    browser
+      .wait(Helper.EC.elementToBeClickable(this.getTableCell(name)), Helper.TIMEOUT)
+      .then(() => {
+        this.getTableCell(name).click();
+      });
+
+    // Clicks the Edit Mode button
+    const editModeButton = element(by.cssContainingText('button', 'Edit Mode'));
+    browser.wait(Helper.EC.elementToBeClickable(editModeButton), Helper.TIMEOUT).then(() => {
+      editModeButton.click();
+    });
+    // Clicks the drop down in the edit pop-up, then clicks the Update button
+    browser.wait(Helper.EC.visibilityOf($('.modal-content')), Helper.TIMEOUT).then(() => {
+      const mirrorDrop = element(by.id('mirrorMode'));
+      browser
```

Comment:
Please use `moveClick`
```suggestion
      this.moveClick
```